### DISCUSSION TITLE
Discussing our presence on X (twitter)

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2662] [Media]
IMO LXQt should stay away from politics, but recently [one of our presences](https://twitter.com/lxqt_project) on social media has become more and more involved in politics.

I see 3 options:

1. Continue and ignoring the environment.
2. Stop actively using it with a message.
3. Closing the account.

Thoughts?

**Comment by tsujan** [comment]
IMHO, 2 may be interpreted as violating the freedom of speech, or even worse.

**Reply by stefonarch** [reply]
Why? With _message_ I intended just 2 words that we would like to stay out of politics and X isn't unpolitical anymore.

**Reply by tsujan** [reply]
I told why ;) But I have no experience in social media.

**Comment by guek** [comment]
Am 20.12.24 um 20:03 schrieb Standreas:
1.
("Leaving with message" would be dumb "activist"'s like politics)
-- 
.

**Reply by stefonarch** [reply]
That is true and a paradoxon not easy to solve. Leaving X is political, staying now eventually too...

**Reply by tsujan** [reply]
It's called a "double bind" is psychology ;) The best action is nonaction: just ignoring them.

**Comment by Gnomasz** [comment]
I didn't bother saying when nobody was asking, but now I'd like to chime in as a user: the politics of X bother me less than the fact that for a couple of months already one has to register and log in to see any tweet. So for a couple of months now I don't read https://twitter.com/lxqt_project. However, as long as there are release notes on the RSS It doesn't bother me, to be fair.

**Reply by stefonarch** [reply]
RSS takes only updates from the website, but sometimes there's more content, https://mastodon.social/@LXQt can be read without account.

**Comment by stefonarch** [comment]
https://micronews.debian.org/2025/1738154246.html

**Reply by guek** [reply]
It's the strange opinion of the "Debian Publicity Team".
Who "don't feel safe" on X may use the other media they mention.
I love Debian software, but I disagree with its Publicity Team.

PS: I love X! :-)

**Reply by tsujan** [reply]
> I love X! :-)

Did anyone tell you that you should hate it?

It was just a quote from Debian.

**Reply by tsujan** [reply]
@stefonarch 
If being on X causes such emotional reactions, I vote for closing the account. No one has time for such things.

**Reply by guek** [reply]
"It was just a quote from Debian."
It was a link.
But why posting if no reply is expected?
Why keep the anti-𝕏 campaign going? Don't be insincere, please.

Please, stop politisizing Free Software development!

**Reply by stefonarch** [reply]
I posted it here because it is related, and it's not about anti or pro X, it's about not have to be either and if that is possible still.

**Reply by tsujan** [reply]
> Why keep the anti-𝕏 campaign going? Don't be insincere, please.

Haha… even here! Enough is enough.

**Comment by stefonarch** [comment]
Sinking ship? 
https://x.com/Xubuntu/status/1886754277177123276

**Comment by jubalh** [comment]
I'm for option `1`.
I would propose to just post the same messages/news on all social media challenges where we have accounts. And don't react on any hypes and politics and whatever.
It is just a way to put news out to users of LXQt and I don't care where they are. I don't care why they use that social network. I just care that they read the news.

**Reply by stefonarch** [reply]
That's a point, but it's me checking and maintaining our account on X and it doesn't feel like a very nice place lately.

**Reply by jubalh** [reply]
So you are getting rude messages there via PM? If yes, then thats a problem. Maybe we can disable them.
Or you are scrolling through it and then have the impression it's not nice?

I wouldn't use it as a communication platform. Just as a way to push out news. Maybe there an application that let's us post the same thing on multiple social medias? Then one could just write a post push it to Mastodon X and whatever else is there without logging in on each website.

**Reply by stefonarch** [reply]
no, no rude PMs, but I had to silence Musk and sometimes the timeline was full of trumpism stuff. Communication/interaction doesn't happen often and is fine, but it's more on mastodon. 
Another point is that mastodon/rss/website is open to everybody, no registration required.

**Reply by jubalh** [reply]
Well I have no strong opinion about it tbh. I just think we are a relatively small project and its good to reach people at as many places as possible.

**Reply by Frtzqe** [reply]
I agree with **jubalh.** Any presence on social media is nice to promote lxqt-project, but really, it doesn't matter. 

Distros is what matters. Or some noise, or just good training-videos on plain distros that do not advertise us already. How to install LXQt on out of the box Debian, Arch, Void, openSuse, etc.
